plugins/services/src/js/components/modals/ServiceRootGroupModal/index.tsx

```diff
@@ -196,14 +244,44 @@ class ServiceRootGroupModal extends React.Component<
     });
   }
 
+  getGroupFormData(): void {
+    const { id } = this.props;
+    if (!!id) {
+      getGroup(id)
+        .pipe(take(1))
+        .subscribe({
+          next: groupData => {
+            const data = groupFormDataFromGraphql(groupData.data.group);
+            this.setState({
+              data,
+              originalData: JSON.parse(JSON.stringify(data))
+            });
+          },
+          error: () => {
+            this.setState({ error: true });
```

Comment:
can this be `loadingError` or something a bit more descriptive.

Reply:
We can use this for other types of errors as well in the future.